frontend/src/app/_services/index.ts

```diff
@@ -1,2 +1,3 @@
 export * from './account.service';
-export * from './schedule.service';
\ No newline at end of file
+export * from './schedule.service';
+export * from './creator.service';
```

Comment:
hahaha ok yeah

Reply:
i have done this for all the files i have touched since i read this comment. it's now a setting on my editor. we'll fix the rest of them as we go through.